app/models/event.rb

```diff
@@ -15,6 +15,32 @@ class Event < ActiveRecord::Base
 
   validates :max_participants, numericality: { only_integer: true, greater_than: 0 }
 
+  # Returns whether all application_letters are classified or not
+  #
+  # @param none
+  # @return [Boolean] if status of all application_letters is not equal 2
+  def applicationsClassified?
+    application_letters.all? { |application_letter| application_letter.status != 'pending' }
+  end
+
+  # Returns a string of all email addresses of rejected applications
+  #
+  # @param none
+  # @return [String] Concatenation of all email addresses of rejected applications, seperated by ','
+  def compute_rejected_applications_emails
+    rejected_applications = application_letters.select { |application_letter| application_letter.status == 'rejected' }
```

Comment:
I think the difference is that `select` fetches all objects from the database and then iterates over them to select the right ones, whereas `where` uses SQL to and thus does the selection inside the db.